lib/ChangeTypes.js

```diff
@@ -58,11 +64,13 @@ module.exports = {
   // e.g. x^2 * x -> x^2 * x^1
   ADD_EXPONENT_OF_ONE: 'ADD_EXPONENT_OF_ONE',
   // e.g. x^2 * x^3 * x^1 -> x^(2 + 3 + 1)
-  COLLECT_EXPONENTS: 'COLLECT_EXPONENTS',
+  COLLECT_POLYNOMIAL_EXPONENTS: 'COLLECT_POLYNOMIAL_EXPONENTS',
   // e.g. 2x * 3x -> (2 * 3)(x * x)
   MULTIPLY_COEFFICIENTS: 'MULTIPLY_COEFFICIENTS',
   // e.g. 2x * x -> 2x ^ 2
   MULTIPLY_POLYNOMIAL_TERMS: 'MULTIPLY_POLYNOMIAL_TERMS',
+  // e.g x^5 / x^3 -> x^2
+  DIVIDE_POLYNOMIAL_TERMS: 'DIVIDE_POLYNOMIAL_TERMS',
```

Comment:
It is handled by cancelling but I think it is clearer with this change group. Guess it depends on the exercise